common/scala/src/main/scala/org/apache/openwhisk/core/containerpool/logging/SplunkLogStore.scala

```diff
@@ -98,12 +97,22 @@ class SplunkLogStore(
 
   val maxPendingRequests = 500
 
+  def createInsecureSslEngine(host: String, port: Int): SSLEngine = {
```

Comment:
does it make sense to refactor this method into a singleton (it's used in more than one place)?

Reply:
maybe not - only saw it used in two places.

Reply:
yeah, just in one really